apps/backend/src/app/user-assessments-create/user-assessments-write-handler.factory.ts

```diff
@@ -25,6 +27,7 @@ export const userAssessmentsWriteHandlerFactory = (
                 ...assessment,
                 assessmentId: change.after.id,
                 assessor: assessor,
+                verifiedCategories: assessmentCategories?.data() as UserAssessmentVerifiedCategories ?? null,
```

Comment:
a jakby dać ?? {} to spełniłoby interfejs a to od razu poprawny pusty obiekt który ogranicza posługiwanie się undefined/nullem po systemie, może warto wrzucić?